storage/scylla_config.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
```

Comment:
How about consolidating the entries about ScyllaDB into top-level configurations?

Reply:
Is here means a configuration file which defines,

- ScyllaDB entries

- account/password for authentication

- And anything would be changed as Scylla cluster setting change.

Reply:
If we need extra configuration file(s), there must be corresponding document(s).

Reply:
I am wondering the default host, just use another server's IP instead of node0?
To specify to user-defined host, could it simply be an argument when execute?
For example, `bazel test //test:test_scylla --test_arg=HOST`

If we have extra configuration file(s), I think we need a default path or filename and could be config with command too. I study how TA read CLI opt and parse the YAML.
And this commit https://github.com/DLTcollab/tangle-accelerator/commit/55630777ba9d80f8dc418293ea1fa0f1f0fafd6a which also use a configuration file.
I am not sure which method is more proper.